Python/Internshala_Scrapper/internshala.py

```diff
@@ -0,0 +1,93 @@
+# This script will scrape all the internships with given url
+# and create a CSV sheet out of it.
+
+# Necessary imports
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+# A dictionary that will initially hold all scraped values
+scraped_data = {
+    'heading': [],
+    'company': [],
+    'stipend': [],
+    'apply_by': [],
+    'start_date':[],
+
+}
+
+
+url = "https://internshala.com/internships/computer%20science-internship"
+response = requests.get(url)
+data = response.text
+soup = BeautifulSoup(data, 'html.parser')
+count_of_internships = int(soup.find("div",
+                                     class_="heading_4_6").text.split()[0])
+
+num_of_pages = int((count_of_internships / 40) + 1)
+# A loop that will go to each page and will scrape the data
+for i in range(1, num_of_pages + 1):
+    # ------------------- Scraping starts here -------------------------------
+    response = requests.get(f"{url}/page-{0}".format(i))
+    print(response.status_code)  # Check out response whether its 200 or not

```

Comment:
instead of printing Code 200 for sucess, you can add a Success message to make it more user friendly.